templates/database_docker_backup.sh.j2

```diff
@@ -62,7 +77,17 @@ do
   BACKUP_FOLDER=$CURRENT_DIR/backups/$i
   mkdir -p $BACKUP_FOLDER
   docker exec $i sh -c 'mongodump -u "$MONGO_INITDB_ROOT_USERNAME" -p "$MONGO_INITDB_ROOT_PASSWORD" --archive' | zstd > $BACKUP_FOLDER/$(date +%Y%m%d-%H%M).archive.zst
+{% if docker_database_backup_ping | length %}
+  SUM_OF_EXIT_CODES+=${PIPESTATUS[0]}
+  SUM_OF_BACKUPS+=1
+{% endif %}
 done
 
 # delete backups older than 14 days
-/usr/bin/find $CURRENT_DIR/backups/ -mtime +$KEEP_DAYS -delete -type f
+/usr/bin/find $CURRENT_DIR/backups/ -mtime +$KEEP_DAYS -type f -delete
+
+{% if docker_database_backup_ping | length %}
+# ping monitoring
+ping_body="[${SUM_OF_BACKUPS}x] $(tail -n 4 {{ docker_database_backup_path }}/last_run.txt)"
```

Comment:
`[${SUM_OF_BACKUPS}x]` please give some context here. for a user without knowing the script, it is unclear what [2x] means in the log. can we change this to `[${SUM_OF_BACKUPS} backups started]`. a line break below this line would be helpful as well! we think, you need to use `echo` whith piping into a var